test-apps/ui-test-app/.env.template

```diff
@@ -1,19 +1,19 @@
 # ui-test-app config variables
```

Comment:
@calebmshafer should i revert changes to this file if we're not using the `.env` anymore?

Reply:
Ah I must have missed this when fixing everything.  Yes, lets revert it.

Reply:
reverted